src/main/java/karmosin/ResultEmailer.java

```diff
@@ -8,5 +8,5 @@
      * 
      * @param continuousIntegrationJob
      */
-    public void emailResult(ContinuousIntegrationJob continuousIntegrationJob);
+    public boolean emailResult(ContinuousIntegrationJob continuousIntegrationJob);
```

Comment:
please do not modify the interface definitions as this will result in lots of changes to others PRs.

Reply:
changed back to void